packages/js/product-editor/src/blocks/input/block.json

```diff
@@ -0,0 +1,41 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-input-field",
```

Comment:
I think that the name for this block should be `woocommerce/product-text-field`, since it only handles text input.